### ISSUE TITLE
Update documentation on Symphony website

### ISSUE THREAD

**Issue by tschneidereit** [symphonycms/jit_image_manipulation#39]
The [documentation on the website](http://www.getsymphony.com/learn/concepts/view/jit-image-manipulation/) doesn't contain information about the *resize to fit* method. It would be helpful to include this.

**Comment by nitriques** [comment]
Still not fixed. New url is http://www.getsymphony.com/learn/concepts/view/jit-image-manipulation/

**Comment by allen** [comment]
Much apologies. I'm so out of the loop with the changes to JIT it'll take me a bit of time to figure out the changes. If someone is happy enough to provide a quick and dirty pathing reference, I can make the appropriate changes on the website.

**Comment by nitriques** [comment]
@allen The information in the README should be enough... no?

**Comment by allen** [comment]
Yes, it's enough. I spoke to Brendan and got some more info about mode 4 (resize to fit).

I have made the appropriate changes on the website to reflect the updated JIT extension.

Specifically the following was updated:

- Added mode 4 description
- Added Recipes section
- Renamed "External Images" to "External Images & Trusted Sites" to better reflect the behaviour and functionality.

Let me know if I've missed anything.
